tutorial/05 - Presentation Layouts.ipynb

```diff
@@ -362,7 +358,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In the previous chapters we started to learn how to create single plots using differnet kinds of data. But we often want to plot more than one thing. Bokeh plots can be individually embedded in HTML documents, but it's often easier to\n",
+    "In the previous chapters,we started to learn how to create single plots using different kinds of data. But we often want to plot more than one thing. Bokeh plots can be individually embedded in HTML documents, but it's often easier to\n",
```

Comment:
missing comma again. Seems to be a consistent problem, I won't not any future occurrences explicitly but all changes should be checked that commas have spaces after them.